adform-htb.js

```diff
@@ -409,10 +490,10 @@ function AdformHtb(configs) {
                 pm: 'ix_adfrm_cpm',
                 pmid: 'ix_adfrm_dealid'
             },
-            bidUnitInCents: 1, // The bid price unit (in cents) the endpoint returns, please refer to the readme for details
+            bidUnitInCents: 100, // The bid price unit (in cents) the endpoint returns, please refer to the readme for details
             lineItemType: Constants.LineItemTypes.ID_AND_SIZE,
-            callbackType: Partner.CallbackTypes.ID, // Callback type, please refer to the readme for details
-            architecture: Partner.Architectures.SRA, // Request architecture, please refer to the readme for details
+            callbackType: Partner.CallbackTypes.CALLBACK_NAME, // Callback type, please refer to the readme for details
```

Comment:
According to your mock response, a JSON is returned. Please change  `callbackType` to `Partner.CallbackTypes.NONE`

Reply:
How it would behave with this setting`requestType: Partner.RequestTypes.ANY`? As request also can be jsonp.